client/app/reducers/ManageAvailabilityReducer.js

```diff
@@ -0,0 +1,5 @@
+const initialState = {};
```

Comment:
I guess this comes later when there's more understanding about data format of this state, but saying this aloud anyway - we have had a habit of writing initial state as close to real data as possible.